src/components/AutocompleteLocation.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div class="columns">
+    <GmapPlaceInput
+    placeholder='Enter location You are searching for...'
+    v-on:place_changed='placeChange'
+    >
+    </GmapPlaceInput>
+    </div>
+</template>
+
+<script>
+import Store from '../store/store';
+
+export default
+{
+    name: 'autocomplete-location',
+    methods:
+    {
+        placeChange (e)
```

Comment:
"event" instead of "e" -> it'll go through a minificator anyway ;)
[It's short for "event", right?]
Some comment's would be great here